src/vse_sync_pp/analyzers/ppsdpll.py

```diff
@@ -0,0 +1,34 @@
+### SPDX-License-Identifier: GPL-2.0-or-later
+
+"""Analyze ppsdpll log messages"""
+
+from .analyzer import Analyzer
+
+### TODO
+#prtc_b = 40
+#prtc_a = 100
+
+class PhaseOffsetTimeErrorAnalyzer(Analyzer):
+    """Analyze DPLL phase offset time error"""
+    id_ = 'ppsdpll/phase-offset-time-error'
+    parser = 'dpll/phase-offset'
+    cols = ('phaseoffset',)
+    def prepare(self, rows):
+        return ((float(r[0]),) for r in rows)
+    def test(self, data):
+        pho_min = data.phaseoffset.min()
+        pho_max = data.phaseoffset.max()
+        if max(abs(pho_min), abs(pho_max)) < 40:
+            return True
+        return False
+    def explain(self, data):
+        pho_min = data.phaseoffset.min()
+        pho_max = data.phaseoffset.max()
+        return {
+            'units': 'ns',
+            'min': pho_min,
+            'max': pho_max,
+            'range': pho_max - pho_min,
+            'μ̂': data.phaseoffset.mean(),
```

Comment:
is this mu hat prime?

Reply:
NVM its just a stylised mu. I think mean and variability would be better here as it expresses what it actually is.

Reply:
Not necessaritly. Why?

Reply:
It was just the font. Anyway I still think words are better here.

Reply:
I tend to agree  because not everybody consuming this code is gonna map directly mu and sigma to mean and standard deviation. I find reasonable to change it to letters.